socketio/src/client/reconnect.rs

```diff
@@ -0,0 +1,216 @@
+#![allow(unused)]
+use std::{
+    ops::Deref,
+    sync::{Arc, Mutex, RwLock},
+    time::Duration,
+};
+
+pub use super::super::{event::Event, payload::Payload};
```

Comment:
nit: Is it better to use the absolute path here? `super::super` looks a bit odd, but I might be wrong here and this is common practice.

Reply:
I agree, this is IDE auto added. I'll fix it.